test/data/bittorrent-dht-client.js

```diff
@@ -415,7 +415,10 @@ DHT.prototype._put = function (opts, cb) {
         err.address = node.addr;
         errors.push(err);
       }
-      if (--pending === 0) cb(errors, hash);
+      if (--pending === 0) {
+        return cb(errors, hash);
+      }
+      return undefined;
```

Comment:
No need to specify `return undefined` here. All functions in JS implicitly `return undefined`.